### PR TITLE
fix for colors view

### PR DESCRIPTION
**Pull request by willmcgugan**
Fixed colors panel not rendering

### CODE REVIEW

**Comment by rodrigogiraoserrao** [comment]
I don't see any issue with `textual colors` in `main`, so I don't get what this PR is trying to do.

**Comment by rodrigogiraoserrao** [comment]
For me, they scroll the same. At the bottom, they have vertical padding equal to the vertical padding at the top 🤷

Untitled

File: pyproject.toml
Comment (rodrigogiraoserrao) [inline pyproject.toml:1]: Don't you need to `poetry install` and commit the resulting `poetry.lock`?

**Comment by willmcgugan** [comment]
> For me, they scroll the same. At the bottom, they have vertical padding equal to the vertical padding at the top 🤷

You may have this update. It is pushed to PyPi already.

File: pyproject.toml
Comment (willmcgugan) [reply pyproject.toml:1]: At the time, I dont think 0.33.0 was on PyPi. Shouldn't make any difference though.

**Comment by rodrigogiraoserrao** [comment]
> You may have this update. It is pushed to PyPi already.

Oh, but it hasn't been merged yet?
In that case, why not merge it?

**Pull request merged**